WordPress/Docs/PHP/IniSetStandard.xml

```diff
@@ -0,0 +1,36 @@
+<documentation title="Detect incorrect use of the `ini_set()` function.">
+    <standard>
+    <![CDATA[
+      Using ini_set() for alternating PHP settings, as well as other functions with ability to change configuration at runtime of your scripts is discouraged. Changing runtime configuration might break other plugins and themes.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: ini_set() for a possibly breaking setting.">
+        <![CDATA[
+// ini_set() should not be used.
+        ]]>
+        </code>
+        <code title="Invalid: ini_set() for a possibly breaking setting.">
+        <![CDATA[
+ini_set( <em>'short_open_tag'</em>, 'off' );
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+      For some configuration values there are WordPress functions. These are preferred.
```

Comment:
There is only one config setting for which we suggest a WP function alternative.
For all the others, we suggest either a PHP function or using a WP constant.

I've tried to adjust the sentence to better reflect that.

```suggestion
      For some configuration values there are alternative ways available - either via WordPress native functionality of via standard PHP - to achieve the same without the risk of breaking interoperability. These alternatives are preferred.
```